feat: add `ndarray/flatten-from-by` (#9040)
stdlib-js/stdlib: lib/node_modules/@stdlib/ndarray/flatten-from-by/README.md

<!--

@license Apache-2.0

Copyright (c) 2025 The Stdlib Authors.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

   http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.

-->

# flattenFromBy

> > Flatten an [ndarray][@stdlib/ndarray/ctor] from a specified dimension while applying a callback function to each element.

Reviewer: ```suggestion
> Flatten an [ndarray][@stdlib/ndarray/ctor] according to a callback function starting from a specified dimension.
```

Applies here and throughout this PR.
